app/components/dynamic-content/dynamic-content-form/dynamic-content-form-object/dynamic-content-form-object.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+angular.module('bulbs.cms.dynamicContent.form.object', [
+  'lodash'
+])
+  .constant('DIRECTIVE_NAMES_MAP', {
+    text: 'dynamic-content-form-field-text'
+  })
+  .directive('dynamicContentFormObject', [
+    '_', '$compile', 'DIRECTIVE_NAMES_MAP',
+    function (_, $compile, DIRECTIVE_NAMES_MAP) {
+      var DynamicContentFormObjectError = BulbsCmsError.build('<dynamic-content-form-object>');
+
+      return {
+        link: function (scope, element, attrs) {
+          var $form = element.find('form');
+
+          scope.$watch('schema', function () {
```

Comment:
Do we need to `$watch` the schema?

This is now passed directly to the directive scope and won't be changing.